satpy/readers/eum_l2_grib.py

```diff
@@ -33,7 +32,7 @@
 from satpy.readers.eum_base import get_service_mode
 from satpy.readers.fci_base import calculate_area_extent as fci_calculate_area_extent
 from satpy.readers.file_handlers import BaseFileHandler
-from satpy.readers.seviri_base import PLATFORM_DICT, REPEAT_CYCLE_DURATION
+from satpy.readers.seviri_base import PLATFORM_DICT, REPEAT_CYCLE_DURATION, REPEAT_CYCLE_DURATION_RSS
```

Comment:
To avoid confusion (see e.g. comment below), perhaps rename the variables to make clear that these are SEVIRI variables:
```
from satpy.readers.seviri_base import (
    PLATFORM_DICT as SEVIRI_PLATFORM_DICT,
    REPEAT_CYCLE_DURATION as SEVIRI_REPEAT_CYCLE_DURATION,
    REPEAT_CYCLE_DURATION_RSS as SEVIRI_REPEAT_CYCLE_DURATION_RSS
)
```